koap/build.gradle

```diff
@@ -1,28 +1,57 @@
 plugins {
-    id 'org.jetbrains.kotlin.jvm'
+    id 'org.jetbrains.kotlin.multiplatform' version '1.3.72'
     id 'org.jmailen.kotlinter'
+    id 'java'
     id 'jacoco'
-    id 'com.vanniktech.maven.publish'
+//    id 'com.vanniktech.maven.publish' // TODO Move to "maven-publish" ?
```

Comment:
Move into `jvm` block below (as `apply plugin: ...`). Since we only need Maven publication for JVM artifacts it's fine to move it out of the entire module configuration (and into `jvm`).

Reply:
Updated